osu.Game/Overlays/RankingsOverlay.cs

```diff
@@ -9,32 +9,35 @@
 using osu.Game.Overlays.Rankings;
 using osu.Game.Users;
 using osu.Game.Rulesets;
-using osu.Game.Online.API;
 using System.Threading;
 using osu.Game.Graphics.UserInterface;
+using osu.Game.Overlays.Rankings.Displays;
+using System;
 using osu.Game.Online.API.Requests;
-using osu.Game.Overlays.Rankings.Tables;
+using osu.Game.Online.API;
 
 namespace osu.Game.Overlays
 {
     public class RankingsOverlay : FullscreenOverlay
     {
-        protected Bindable<Country> Country => header.Country;
+        public const float CONTENT_X_MARGIN = 50;
+
+        protected readonly Bindable<Country> Country = new Bindable<Country>();
```

Comment:
let's put it this way: ideally I'd like for this to not exist here. the rankings overlay should not need to know what country its sub-view is displaying.